### PR TITLE
[TerritorialCouncil][Referent] Fix qualities display when filtering by quality + leader quality

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: templates/referent/territorial_council/members.html.twig [hunk 1]
```diff
@@ -150,13 +150,20 @@
...
+                                        {% set pcQualities = adherent.politicalCommitteeMembership ? adherent.politicalCommitteeMembership.qualityNames : [] %}
                                         {% for quality in membership.qualities %}
                                             <b>{{ ('territorial_council.membership.quality.' ~ quality.name)|trans }}</b> {{ quality.zone }}
 
                                             {% if adherent.findMandatesForQuality(quality.name, true)|length %}
                                                 <span class="elected-tag elected-tag--small text--on-yellow text--bold b__nudge--left-nano">désigné{{ adherent.isFemale ? 'e' }}</span>
                                             {% endif %}
                                             <br />
+                                            {% if pcQualities %}
+                                                {% set pcQualities = pcQualities|filter((value) => value != quality.name) %}
```
Comment (kevin-verschaeve) [inline templates/referent/territorial_council/members.html.twig:162]: is it normal that this variable can be modified multiple times before getting used ?

Comment (illusionOfParadise) [reply templates/referent/territorial_council/members.html.twig:162]: Yes, I want to display only PC qualities which don't correspond to TC qualities, that's why they are removed from the variable.

Comment (ottaviano) [reply templates/referent/territorial_council/members.html.twig:162]: what about improving it a little bit: 
```twig 
{% set pcQualities = adherent.politicalCommitteeMembership ? adherent.politicalCommitteeMembership.qualityNames : [] %}

{% for quality in membership.qualities %}
  // ...
{% endfor %}

{% for pcQuality in pcQualities|filter(quality => quality not in membership.qualities) %}
  // ...
{% endfor %}

```

**Comment by illusionOfParadise** [comment]
[ch12538]

**Review by ottaviano — APPROVED** [review]

**Pull request merged**